README.md

```diff
@@ -31,14 +31,14 @@ The project is nowhere near usable. The code written so far is published for dev
 
 * Download this git repository. Open the file `src/main.rs` and edit the RPC username and password in the function `get_bitcoin_rpc`. Make sure your Bitcoin Core has a wallet called `teleport`, or edit the name in the same function.
 
-* Create two teleport wallets by running `cargo run generate-wallet` twice. Call them something like `maker.teleport` and `taker.teleport`.
+* Create two teleport wallets by running `cargo run -- --wallet_file_name=<wallet-name> generate-wallet` twice. Instead of `<wallet-name>`, use something like `maker.teleport` and `taker.teleport`.
 
-* Use `cargo run get-receive-invoice maker.teleport` to obtain 3 addresses of the maker wallet, and send regtest bitcoins to each of them (amount 5000000 satoshi or 0.05 BTC in this example). Do this as well for the `taker.teleport` wallet. Get the transactions confirmed.
+* Use `cargo run -- --wallet_file_name=maker.teleport get-receive-invoice` to obtain 3 addresses of the maker wallet, and send regtest bitcoins to each of them (amount 5000000 satoshi or 0.05 BTC in this example). Do this as well for the `taker.teleport` wallet. Get the transactions confirmed.
 
-* Check your balance with `cargo run wallet-balance maker.teleport`. Example:
+* Check your balance with `cargo run -- --wallet_file_name=maker.teleport wallet-balance`. Example:
```

Comment:
This should use dashes not underscores, like `--wallet-file-name`. The same thing appears in couple of other places in the readme

Reply:
Good catch, fixed in 4f57bae730516d74a7e3fe04947620dc3541afff.